test/golden/library_example.golden

```diff
@@ -31,6 +31,23 @@
 /gapic-google-cloud-library_example-v1/tests/unit/gapic/v1/test_library_service_client_v1.py
 /gapic-google-cloud-library_example-v1/tox.ini
 /gcloud-php-cloud-library_example/composer.json
+/gcloud-php-cloud-library_example/proto/src/GPBMetadata/Google/Example/Library/V1/Library.php
```

Comment:
Good point - after the [googleapis PR associated with this PR](https://github.com/googleapis/googleapis/pull/418) is merged, this will no longer be consistent. Updated